xml/common_intro_available_doc_i.xml

```diff
@@ -41,27 +41,13 @@
 
  <variablelist>
   <varlistentry os="sles;sled">
-   <term><xref linkend="book.cap.deployment"/>
+   <term><xref linkend="book.cap.guides"/>
    </term>
    <listitem>
     <para>
-     The &productname; deployment guide gives you details about installation
-     and configuration of &productname; along with a description of
-     architecture and minimum system requirements.
+     The &productname; guide is a comprehensive guide providing deployment, administration, and user guides, and      architecture and minimum system requirements.
```

Comment:
There are some extra spaces between "and" and "architecture"

Reply:
I will fix that and run daps-xmlformat in a separate commit.